test/cpp/app_seat_service_client/app/AppManifest.json

```diff
@@ -15,6 +15,22 @@
                 "src": "bcm/horn/v1/horn.proto",
                 "required": {}
             }
+        },
+        {
+            "type": "grpc-interface",
+            "config": {
+              "src": "https://github.com/eclipse-kuksa/kuksa-databroker/archive/refs/tags/0.4.5.zip",
+              "pathInZip": "kuksa-databroker-0.4.5/proto/kuksa",
+              "protoIncludeDir": "0.4.5/kuksa-databroker-0.4.5/proto",
```

Comment:
Like above, the include dir should be relative to the "src". If it is a zip archive, it should be relative to the extracted directory and not include it in its path.